app/src/main/java/de/westnordost/streetcomplete/quests/crossing_type/AddCrossingTypeForm.java

```diff
@@ -0,0 +1,42 @@
+package de.westnordost.streetcomplete.quests.crossing_type;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.quests.ImageListQuestAnswerFragment;
+
+public class AddCrossingTypeForm extends ImageListQuestAnswerFragment
+{
+    private final OsmItem[] TYPES = new OsmItem[] {
+            new OsmItem("traffic_signals", R.drawable.fire_hydrant_pillar, R.string.quest_crossing_type_signals),
+            new OsmItem("uncontrolled", R.drawable.fire_hydrant_pillar, R.string.quest_crossing_type_uncontrolled),
+            new OsmItem("island", R.drawable.fire_hydrant_pillar, R.string.quest_crossing_type_island),
+            new OsmItem("unmarked", R.drawable.fire_hydrant_pillar, R.string.quest_crossing_type_unmarked)
```

Comment:
I can't find icons that could describe those answers... I would prefer images, but I don't know how to import them. I tried the Android Studio Image Asset Generator but that doesn't work...